File: Backend/src/main/java/com/feedle/feedleapi/Networking/AddConversationResponse.java
package com.feedle.feedleapi.Networking;

import com.feedle.feedleapi.Models.UserConversation;

import java.util.ArrayList;

public class AddConversationResponse extends Request {

    private ArrayList<UserConversation> userConversations;

    public AddConversationResponse(ArrayList<UserConversation> userConversations) {
        super(RequestType.AddConversationResponse);
        this.userConversations = userConversations;
    }

    public ArrayList<UserConversation> getUserConversations() {
        return userConversations;
    }
}

File: FeedleDataTier/FeedleDataTier/Network/AddConversationResponse.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class AddConversationResponse : Request
    {
        [JsonPropertyName("userConversations")]
        public List<UserConversation> UserConversations { get; set; }

        public AddConversationResponse(List<UserConversation> userConversations) : base(RequestType.AddConversationResponse)
        {
            this.UserConversations = userConversations;
        }
    }
}